Makefile

```diff
@@ -59,7 +59,13 @@ test: run-unit-tests
 
 .PHONY: lint
 lint:
-	golangci-lint run --enable-all --disable=godox,gochecknoglobals --max-same-issues=0 --max-issues-per-linter=0 --build-tags aws,packet,e2e,disruptive-e2e --new-from-rev=$$(git merge-base $$(cat .git/resource/base_sha 2>/dev/null || echo "master") HEAD) --modules-download-mode=$(MOD) --timeout=5m --exclude-use-default=false ./...
+	git remote add linter https://github.com/kinvolk/lokomotive.git || true
```

Comment:
I'm not sure if this is optimal, but that makes sure we always lint against upstream master branch.

Reply:
What we want almost always is "origin/master" at that is all.

It will fail in the cases that upstream is not origin? Sure. We can document it and be happy, IMHO

Reply:
Well, if you fork the repository, usually `origin` points to the fork and as far as I know, not everyone keeps their fork master up to date, hence comparison with upstream.

Reply:
I removed it, as it's causing troubles when remote is added via Docker (uid fun).